Changing the default `PGPORT` settings breaks the flow

Currently it is impossible to use the OMT flow if the value of `PGPORT` is modified in `.env`.

## To reproduce:
1. Modify `.env` to use `PGPORT=54321`
2. Run `make start-db`

```
$ make start-db
Starting postgres docker compose target using default image (no recreate if exists)
Creating master_postgres_1 ... done
Wait for PostgreSQL to start...
docker-compose run --rm --user=1000:1000 openmaptiles-tools pgwait
Creating master_openmaptiles-tools_run ... done
... gave up waiting for Postgres:   PGHOST=postgres PGDATABASE=openmaptiles PGUSER=openmaptiles PGPORT=54321 pg_isready
ERROR: 1
make: *** [Makefile:306: start-db] Error 1
```

## Analysis
Port `5432` is always used by the `openmaptiles/postgress` container. As a result, all other containers need to access the PGDB using the `5432` port.

## Alternative solutions:
1. Remove the `PGPORT` from the `.env` file and replace all other occurrences of `PGPORT` with hard-code `5432` in the `openmaptiles` and `openmaptiles-tools` repositories.
2. Modify `openmaptiles/postgress` Docker image to consider `PGPORT` and start the DB using `postgress -p ${PGPORT:-5432}`

--- a/.env
+++ b/.env
@@ -4,7 +4,7 @@
 TILESET_FILE=openmaptiles.yaml
 
 # Use 3-part patch version to ignore patch updates, e.g. 5.0.0
-TOOLS_VERSION=6.1
+TOOLS_VERSION=latest
 
 # Make sure these values are in sync with the ones in .env-postgres file
 PGDATABASE=openmaptiles

--- a/.env-postgres
+++ b/.env-postgres
@@ -1,8 +1,0 @@
-# This file defines environment variables for the PostgreSQL image.
-# The main docker PostgreSQL image requires these vars rather than
-# the standard PG* ones that all PostgreSQL tools use.
-
-# Make sure these values are in sync with the ones in .env file
-POSTGRES_DB=openmaptiles
-POSTGRES_USER=openmaptiles
-POSTGRES_PASSWORD=openmaptiles

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -18,8 +18,14 @@ services:
     networks:
       - postgres
     ports:
-      - "5432"
-    env_file: .env-postgres
+      - "${PGPORT:-5432}:${PGPORT:-5432}"
+    env_file: .env
+    environment:
+      # postgress container uses old variable names
+      POSTGRES_DB: ${PGDATABASE:-openmaptiles}
+      POSTGRES_USER: ${PGUSER:-openmaptiles}
+      POSTGRES_PASSWORD: ${PGPASSWORD:-openmaptiles}
+      PGPORT: ${PGPORT:-5432}
 
   import-data:
     image: "openmaptiles/import-data:${TOOLS_VERSION}"
@@ -44,6 +50,10 @@ services:
       IMPOSM_CONFIG_FILE: ${IMPOSM_CONFIG_FILE}
       # Control import-sql processes
       MAX_PARALLEL_PSQL: ${MAX_PARALLEL_PSQL}
+      PGDATABASE: ${PGDATABASE:-openmaptiles}
+      PGUSER: ${PGUSER:-openmaptiles}
+      PGPASSWORD: ${PGPASSWORD:-openmaptiles}
+      PGPORT: ${PGPORT:-5432}
     networks:
       - postgres
     volumes:
@@ -71,6 +81,10 @@ services:
       MBTILES_NAME: ${MBTILES_FILE}
       # Control tilelive-copy threads
       COPY_CONCURRENCY: ${COPY_CONCURRENCY}
+      PGDATABASE: ${PGDATABASE:-openmaptiles}
+      PGUSER: ${PGUSER:-openmaptiles}
+      PGPASSWORD: ${PGPASSWORD:-openmaptiles}
+      PGPORT: ${PGPORT:-5432}
 
   generate-vectortiles:
     image: "openmaptiles/generate-vectortiles:${TOOLS_VERSION}"
@@ -88,6 +102,10 @@ services:
       # Control tilelive-copy threads
       COPY_CONCURRENCY: ${COPY_CONCURRENCY}
       #
+      PGDATABASE: ${PGDATABASE:-openmaptiles}
+      PGUSER: ${PGUSER:-openmaptiles}
+      PGPASSWORD: ${PGPASSWORD:-openmaptiles}
+      PGPORT: ${PGPORT:-5432}
 
   postserve:
     image: "openmaptiles/openmaptiles-tools:${TOOLS_VERSION}"

